shop_stripe/payment.py

```diff
@@ -22,8 +24,12 @@ def get_payment_request(self, cart, request):
         """
         From the given request, add a snippet to the page.
         """
+        stripe.api_key = settings.SHOP_STRIPE['APIKEY']
+        stripe.api_version = settings.SHOP_STRIPE_API_VERSION
```

Comment:
Did we add this setting?  Should we put it in the SHOP_STRIPE dictionary?  (SHOP_STRIP['API_VERSION'])  I could be convinced against this if the original authors ever (or might?) iterate the dictionary.  Debate internally, happy with either decision.